### ISSUE TITLE
cpplint does not detect space before parens

### ISSUE THREAD

**Issue by janusw** [google/styleguide#62]
Consider the following C++ code:

``` C++
int cg() {
  return 0;
}

int main() {
  int i = cg ();  // extra space before parens detected
  return cg ();   // not detected
}
```

cpplint correctly detects the space before parens in the first function call

```
parens.cpp:6:  Extra space before ( in function call  [whitespace/parens] [4]
```

but unfortunately misses the second one.

**Comment by vapier** [comment]
public cpplint development in this repository has been halted (see #837).  please migrate to https://github.com/cpplint/cpplint now.
